### DISCUSSION TITLE
Any plans to implement the ProjectItemsEvents?

### DISCUSSION THREAD

**Discussion by sboulema** [VsixCommunity/Community.VisualStudio.Toolkit#90] [Q&A]
Any plans to also expose the ProjectItemsEvents that are available in the DTE?

https://docs.microsoft.com/en-us/dotnet/api/envdte.projectseventsclass.itemadded?view=visualstudiosdk-2019

Would be really helpful to have the ItemAdded, ItemRenamed and ItemRemoved events.

**Comment by madskristensen** [comment]
Yes, there's an issue on it https://github.com/VsixCommunity/Community.VisualStudio.Toolkit/issues/88
